### ISSUE TITLE
Allow exposing Galene on a specific path (host.com/some/path/galene) on a reverse proxy.

### ISSUE THREAD

**Issue by aguaviva** [jech/galene#178]
This would require a setting for a custom root path that when not set would default to "/". 

Most of us already have nginx working on port 440 and we need to use and open a different port for Galene. This would allows exposing Galene through a reverse proxy at a chosen path)

If I figure out how to do it would you accept a PR?

**Comment by jech** [comment]
> If I figure out how to do it would you accept a PR?

Yes, I would.  Please don't add a command-line option, but a new field in `config.json`.  Some hints:
  * you'll need to change the endpoint advetised in status.json (see the function `group.GetStatus`);
  * you'll need to work out how the new feature interacts with `proxyURL` and with `canonicalHost`.

There's nothing else off the top of my head, I'll update this comment if something comes to mind.

**Comment by jech** [comment]
The server-side code was done in 29e006037c915aac80648604498aae1fc3926265.  The client code needs to adapted, and I'm not sure how to do that automatically, for now you'll need to adapt it manually.

Closing for now, please reopen if you have an idea about how to improve this.
